### PR TITLE
Multiclass assignment example

### PR DESCRIPTION
**Pull request by pedrocamargo**

### CODE REVIEW

File: docs/source/examples/plot_toll_analysis.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+truckDemand.computational_view('trucks')
+truckClass = TrafficClass(truckGraph, truckDemand)
+truckClass.set_pce(1.5)
+truckClass.set_vot(35)
+truckClass.set_fixed_cost('toll', 0.05)
+# %%
+
+
+assig = TrafficAssignment()
+# Different (and better) than some commercial software, your results are as proportional as the
```
Comment (janzill) [inline docs/source/examples/plot_toll_analysis.py:84]: do we want to remove the first part of the sentence?

Comment (pedrocamargo) [reply docs/source/examples/plot_toll_analysis.py:84]: Not really....  :-)

Comment (pedrocamargo) [reply docs/source/examples/plot_toll_analysis.py]: Decided to change. You were right, @janzill

**Comment by janzill** [comment]
lgtm

**Review by janzill — APPROVED** [review]

File: docs/source/examples/plot_toll_analysis.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+stdout_handler.setFormatter(formatter)
+logger.addHandler(stdout_handler)
+
+# %%
+
+disk_pth = join(gettempdir(), "sioux_falls_multi-Class")
+urllib.request.urlretrieve("https://www.aequilibrae.com/data/sioux_falls_multi-Class.zip", disk_pth + ".zip")
+
+with zipfile.ZipFile(disk_pth + ".zip") as zf:
+    zf.extractall(gettempdir())
```
Comment (TTalex) [inline docs/source/examples/plot_toll_analysis.py:35]: This extracts files to the '/tmp' folder on my machine, it doesn't put them in '/tmp/sioux_falls_multi-Class/', so it fails on the following line `project.open(disk_pth)` (the folder doesn't exist).
Should it be `zf.extractall(disk_pth)` instead ?

**Pull request closed without merging**